extension/src/experiments/index.ts

```diff
@@ -568,7 +571,7 @@ export class Experiments extends BaseRepository<TableData> {
           AvailableCommands.GIT_GET_NUM_COMMITS,
           this.dvcRoot
         ),
-      () => this.cliData.update()
+      (...args: (ExperimentFlag | string)[]) => this.cliData.update(...args)
```

Comment:
[I] There is a bug here where any update created by the watchers will use the `-n <number-of-commits>` call. Leaving the webview out of sync with the data being displayed:


https://user-images.githubusercontent.com/37993418/229643446-20953424-1c76-40e0-b52c-79f7565ad699.mov

There are a couple of options but I think the best is to add the `experimentsModel` into `experimentsData` and use both `model.getNbOfCommitsToShow()` and `model.getIsBranchesView()` in the update function

Reply:
Need to fix this one before we 🚢. 

🙏🏻.

Reply:
Totally make sense. Should be fixed now